### ISSUE TITLE
Install fails with add-in prometheus `0.52.0-22.0.0`

### ISSUE THREAD

**Issue by BretFisher** [replicatedhq/kURL#2453]
When Prometheus addon is set to the latest specific version, install fails on Ubuntu 20.04 with error: 

`The CustomResourceDefinition "prometheuses.monitoring.coreos.com" is invalid: metadata.annotations: Too long: must have at most 262144 bytes`

If you set to the previous version listed for prometheus in kurl.sh, it works fine:

```yaml
  prometheus: 
    version: "0.49.0-17.1.3"
```

The below kurl config gives the above error and never completes:

```yaml
apiVersion: "cluster.kurl.sh/v1beta1"
kind: "Installer"
metadata: 
  name: "8d8d24d"
spec: 
  kubernetes: 
    version: "latest"
  weave: 
    version: "latest"
  contour: 
    version: "latest"
  prometheus: 
    version: "0.52.0-22.0.0"
  registry: 
    version: "latest"
  containerd: 
    version: "latest"
  velero: 
    version: "latest"
  ekco: 
    version: "latest"
  fluentd: 
    version: "latest"
  longhorn: 
    version: "latest"
```

**Comment by aj-jester** [comment]
This has been reverted for now. Thanks for reporting! Going forward will recommending pinning _all_ addons to avoid similar issues.

https://github.com/replicatedhq/kURL/pull/2450

**Comment by DanStough** [comment]
Hi @BretFisher, we released 0.53.1-30.1.0 of prometheus, which resolved this issue.
